deutsches-archaologisches-institut-romisch-germanische-kommission.csl

```diff
@@ -0,0 +1,301 @@
+    </author>
+    <category citation-format="author-date"/>
+    <category field="anthropology"/>
+    <category field="humanities"/>
+    <summary>This citation style uses the citation guidelines provided by the Romano-Germanic Commission of the German Archaeological Institute (DGUF). An older version was known as "RGK_archaeology_DGUF". Original external style for reference: https://www.dguf.de/fileadmin/user_upload/Arbeitskreise/AK_DGUF-Zotero/RGK_archaeology_DGUF.csl</summary>
+    <updated>2025-10-15T12:50:00+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="de">
+    <terms>
+      <term name="accessed">zuletzt aktualisiert am</term>
+    </terms>
+  </locale>
+  <macro name="editor">
+    <names variable="editor">
+      <name initialize-with=". " delimiter-precedes-last="always" delimiter="/"/>
+      <label form="short" prefix=" (" suffix=".)" strip-periods="true"/>
+    </names>
+  </macro>
+  <macro name="author">
+    <names variable="author">
+      <name initialize-with=". " delimiter-precedes-last="always" delimiter="/"/>
+      <label form="short" prefix=" (" suffix=".)" strip-periods="true"/>
+      <substitute>
+        <names variable="editor"/>
+        <text variable="title"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="author-sort">
+    <names variable="author">
+      <name name-as-sort-order="all" sort-separator=", " initialize-with="." delimiter-precedes-last="always" delimiter="/"/>
+      <substitute>
+        <names variable="editor"/>
+        <text variable="title"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="author-short">
+    <choose>
+      <if variable="author">
+        <names variable="author">
+          <name form="short" delimiter="/" delimiter-precedes-last="always" et-al-min="3" et-al-use-first="1"/>
+        </names>
+      </if>
+      <else>
+        <names variable="editor">
+          <name form="short" delimiter="/" delimiter-precedes-last="always" et-al-min="3" et-al-use-first="1"/>
+        </names>
+      </else>
+    </choose>
+  </macro>
+  <macro name="container-author">
+    <names variable="container-author">
+      <name initialize-with=". " delimiter-precedes-last="always" delimiter="/"/>
+      <label form="short" prefix=" (" suffix=".)" strip-periods="true"/>
+    </names>
+  </macro>
+  <macro name="access">
+    <text variable="URL" prefix="&lt;" suffix="&gt;"/>
+    <group prefix=" (" delimiter=" " suffix=")">
+      <date variable="accessed">
+        <date-part name="day" form="numeric-leading-zeros" suffix="."/>
+        <date-part name="month" form="numeric-leading-zeros" suffix="."/>
+        <date-part name="year" form="long"/>
+      </date>
+    </group>
+  </macro>
+  <macro name="webpage-date">
+    <group prefix=", " delimiter=" " suffix=". ">
```

Comment:
Generally, we'd prefer if you set these prefixes wherever you call the macros, rather than inside the macros.
If you can rework the whole style in regards to that it'll make your and our life easier maintaining this style.